cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1211,6 +1211,20 @@
       public static final String SSL_TRUSTSTORE_PASSWORD = "security.auth.server.ssl.truststore.password";
     }
 
+    /**
+     * Configurations for setting Twill-related identity.
+     */
+    public static final class TwillIdentity {
+      /**
+       * User identity for Twill runnables which execute user code.
+       */
+      public static final String IDENTITY_USER = "security.twill.identity.user";
```

Comment:
Typically we name class / constant in the order of the configuration, from the generic to specific scope. E.g. in this case, I would prefer something like:

```java
public static final class Twill {
  public static final class Security {
    public static final String IDENTITY_USER = "twill.security.identity.user"
  }
}
```

Reply:
Fixed